Makefile

```diff
@@ -16,6 +16,9 @@ test-unit:
 test-system: build
 	@go test -v -tags=system ./tests -args PATH=$(shell pwd)/dist
 
+test-docs: build
+	@go test -v -tags=system,docs ./tests -run TestDocExamples -args PATH=$(shell pwd)/dist
```

Comment:
A few command-line variants that might be of interest:

- You can select a specific test x to run by doing `-run TestDocExamples/x`. For example: `go test -v -tags=system,docs ./tests -run TestDocExamples/value-match -args PATH=./dist`.

- The default `go test` output format is intended for human consumption. There's a json format that can be abled by adding the `-json` flag to the `go test` invocation, in case you ever wanted to script something around this.